keps/sig-node/1972-kubelet-exec-probe-timeouts/README.md

```diff
@@ -79,11 +79,17 @@ Changes to kubelet:
 E2E tests:
 * re-enable [existing exec liveness probe e2e test](https://github.com/kubernetes/kubernetes/blob/ea1458550077bdf3b26ac34551a3591d280fe1f5/test/e2e/common/container_probe.go#L210-L227) that is currently being skipped
 * add new exec readiness probe e2e test.
+* exec probe tests are promotes to Conformance ([#97619](https://github.com/kubernetes/kubernetes/pull/97619)).
 
 ### Graduation Criteria
 
 This is a bug fix so the feature gate will be GA and on by default from the start.
 
+The feature flag should be kept available till we get a sufficient evidence of people not being
```

Comment:
Should we at least mention some tentative release targets for this?

Reply:
added 1.22 as tentative timeline